torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3213,6 +3220,11 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            dtypesIfCPU=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
            dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
            sample_inputs_func=sample_inputs_masked_select),
+    OpInfo('matrix_exp',
+           dtypesIfCPU=floating_and_complex_types_and(torch.bfloat16),
+           dtypesIfCUDA=floating_and_complex_types_and(torch.bfloat16),
```

Comment:
CUDA needs `torch.float16`, not `torch.Bfloat16`, notice the missing 'b'

Reply:
got it, thanks!